### PR TITLE
Feature/VA-20 hybrid mode

### PR DESCRIPTION
**Pull request by alimd**
Switch back to polymer 1.0 hybrid mode 😢

### CODE REVIEW

File: src/box-menu-icon.html [hunk 1]
```diff
@@ -16,9 +16,9 @@
     </style>
 
     <div>
-      <paper-icon-button on-tap="fireevent" id="one" icon="help-outline"></paper-icon-button>
-      <paper-icon-button on-tap="fireevent" id="two" class="rotate" icon="build"></paper-icon-button>
-      <paper-icon-button on-tap="fireevent" id="three" icon="brush"></paper-icon-button>
+      <paper-icon-button on-tap="fireEvent" id="one" icon="help-outline"></paper-icon-button>
```
Comment (salmanclever) [inline src/box-menu-icon.html:19]: Not work!

File: src/box-menu-icon.js [hunk 2]
```diff
@@ -1,40 +1,10 @@
...
-
-    // threetap () {
-    //   this.fireevent(3);
-    // }
-
-    fireevent (event) {
+  Polymer({
+    is : 'box-menu-icon',
 
+    fireEvent: function (event) {
```
Comment (salmanclever) [inline src/box-menu-icon.js:7]: Not work!

File: src/volcano-app.js [hunk 3]
```diff
@@ -1,76 +1,65 @@
...
       //TODO: add passive polyfill
-    }
+    },
 
-    _onHeaderScroll () {
+    _onHeaderScroll: function () {
       let y = window.scrollY;
       y *= 1.5;
 
       // header background scroll effect
```
Comment (salmanclever) [inline src/volcano-app.js:38]: in opera work but still not work on chrome!
we don't have header shadow in chrome (In linux environment )

**Review by salmanclever — COMMENTED** [review]
It's work in opera and chrome correctly ! but still not work on firefox ?! (in linux environment)

File: src/box-menu-icon.html (hunk 1 above)
Comment (alimd) [reply src/box-menu-icon.html:19]: fixed

**Comment by alimd** [comment]
fixed

**Pull request merged**